src/main/java/nl/tudelft/ewi/devhub/server/database/entities/Assignment.java

```diff
@@ -97,50 +98,61 @@
     private String summary;
 
     @Nullable
-    @Column(name="due_date")
+    @Column(name = "due_date")
     @Temporal(TemporalType.TIMESTAMP)
     private Date dueDate;
 
-    @Column(name="released")
+    @Column(name = "released")
     private boolean gradesReleased;
 
-	@OrderBy("id ASC")
-	@JsonManagedReference
-	@OneToMany(mappedBy = "assignment", cascade = CascadeType.ALL, orphanRemoval = true, fetch = FetchType.LAZY)
-	private List<Task> tasks;
+    @OrderBy("id ASC")
+    @JsonManagedReference
+    @OneToMany(mappedBy = "assignment", cascade = CascadeType.ALL, orphanRemoval = true, fetch = FetchType.LAZY)
+    private List<Task> tasks;
+
+    @OneToMany(mappedBy = "assignment")
+    private List<AssignedTA> assignedTAS;
+
+    public Optional<User> getAssignedTA(Delivery delivery) {
+        Group group = delivery.getGroup();
+        return assignedTAS.stream()
+                .filter(a -> a.getGroup().equals(group))
+                .findAny()
+                .map(AssignedTA::getTeachingAssistant);
+    }
 
     @Override
     public int compareTo(Assignment o) {
         return ComparisonChain.start()
-            .compare(getDueDate(), o.getDueDate(), Ordering.natural().nullsFirst())
-            .compare(getName(), o.getName())
-			.compare(getCourseEdition(), o.getCourseEdition())
-			.compare(getAssignmentId(), o.getAssignmentId())
-            .result();
+                .compare(getDueDate(), o.getDueDate(), Ordering.natural().nullsFirst())
+                .compare(getName(), o.getName())
+                .compare(getCourseEdition(), o.getCourseEdition())
+                .compare(getAssignmentId(), o.getAssignmentId())
+                .result();
+    }
+
+    public double getNumberOfAchievablePoints() {
+        return getTasks().stream()
+                .mapToDouble(Task::getMaximalNumberOfPoints)
+                .sum();
     }
 
-	public double getNumberOfAchievablePoints() {
-		return getTasks().stream()
-			.mapToDouble(Task::getMaximalNumberOfPoints)
-			.sum();
-	}
-
-	@JsonIgnore
-	public GradingStrategy getGradingStrategy() {
-		return new DutchGradingStrategy();
-	}
-
-	@Override
-	public URI getURI() {
-		return getCourseEdition().getURI().resolve(ASSIGNMENTS_PATH_BASE).resolve(getAssignmentId() + "/");
-	}
-
-	public List<Task> copyTasksFromOldAssignment(Assignment oldAssignment) {
-		return oldAssignment.getTasks().stream().map(this::taskforNewAssginment).collect(Collectors.toList());
-	}
-
-	private Task taskforNewAssginment(Task oldTask) {
-		return oldTask.copyForNextYear(this);
-	}
+    @JsonIgnore
```

Comment:
Try to avoid reformatting big unchanged chunks of code. They can cause a lot of difficulty when merging pull requests or figuring out what change introduced a particular problem. Usually its best to revert the change if you see auto formatting was applied prior to committing.